question_answering_service/example_data/data.csv

```diff
@@ -0,0 +1,137 @@
+Step 3: Deploy the REST API Proxy,"Deploying the REST API proxy makes it invokable. By default, Choreo maintains two environments for development and production. Learn more. For this tutorial, let's deploy the proxy to the development environment.In the left navigation menu, click Deploy.
+ Click Configure & Deploy. The Configure & Deploy pane opens.
+ In the Endpoint field, enter https://samples.choreoapps.dev/company/hr.
+ The above URL is the endpoint of the sample API.
+ If you have a sandbox environment to test the API proxy, you can give the URL to that environment in the Sandbox Endpoint field.
+ Click Save & Deploy.Once you have deployed it to the development environment, you can test it. When you are ready to take it to production, you can return to this view and promote it by clicking Promote on the Development card."
+Step 2: Build and deploy,"To implement the REST API, you need to build and deploy it. When you trigger the build you execute the commands in the selected Dockerfile to apply the required configurations to the REST API.To build and deploy the REST API, follow these steps:On the Deploy page, click Build and Deploy.
+ !!! note
+ Deploying the REST API component may take a while. You can track the progress by observing the logs.
+ Once the deployment is complete, the deployment status changes to Active. The Invoke URL field displays the URL via which you can invoke the REST API.
+ Check the deployment progress by observing the console logs on the right of the page.
+ You can access the following scans under Build.
+ The Dockerfile scan: This scans the Dockerfile for a user ID and fails the build if no user is assigned to the Docker container.
+ Container (Trivy) vulnerability scan: This detects vulnerabilities in the Dockerfile-based image and in the third-party packages that the source code uses. If you click Container (Trivy) vulnerability scan, the details of the vulnerabilities open in a separate pane. If this scan detects critical vulnerabilities, the build will fail.
+ !!! info
+ If you want to bypass these vulnerabilities, you need to run Choreo on a private data plane.You can test, manage, and observe this REST API you created from a Dockerfile the way you would a Ballerina component.For detailed instructions, see the following sections of the Create Your First REST API tutorial.Step 3: Test
+ Step 4: Manage"
```

Comment:
new line

Reply:
Addressed in [a12c2db](https://github.com/JayaniH/openai-usecases/pull/1/commits/a12c2dbc9cf69caf2f70c9968b50099a34c52547)